src/main/java/net/robinfriedli/aiode/boot/tasks/VersionUpdateAlertTask.java

```diff
@@ -100,7 +100,7 @@ private void sendUpdateAlert(Context context, Version versionElem, JDA shard) {
             }
 
             List<Guild> guilds = shard.getGuilds();
-            long delaySecs = OFFSET++ * (guilds.size() / MESSAGES_PER_SECOND);
+            long delaySecs = (long) OFFSET++ * ( (long) guilds.size() / MESSAGES_PER_SECOND);
```

Comment:
This seems to be an exhaustive solution to the problem that's been thoroughly presented by SonarQube. I would consider keeping the spatial formatting symmetrical on both sides of the rightmost parenthesis for readability purposes.

```suggestion
            long delaySecs = (long) OFFSET++ * ((long) guilds.size() / MESSAGES_PER_SECOND);
```